codecarbon/emissions_tracker.py

```diff
@@ -279,6 +279,7 @@ def track_emissions(
     offline: bool = False,
     country_iso_code: Optional[str] = None,
     region: Optional[str] = None,
+    gpu_ids: Optional[List] = None,
```

Comment:
can we also update the documentation here https://github.com/mlco2/codecarbon/blob/master/docs/edit/parameters.rst
for the new input param for both the decorator and class constructors